gapic/src/main/com/google/gapid/util/Logging.java

```diff
@@ -123,6 +135,7 @@ public static void setListener(Runnable listener) {
    */
   public static void logMessage(Log.Message message) {
```

Comment:
This is now very much round-about: Buffer.publish calls Logging.logMessage, which calls Buffer.add and then fires the event.

Better to keep it they way it was:
logMessage and Buffer.publish call Buffer.add
Buffer.add fires the event